ex11/index.js

```diff
@@ -0,0 +1,20 @@
+const prompt = require("prompt-sync")({ sigint: true});
+
+console.log("Insert an integer number.");
+const value = parseInt(prompt(">"));
+
+while (value > 0) {
+    const divisor = 2;
+
+    if (value % divisor === 0 ) {
+        console.log(`${value} is an even number.`);
+    }else {
+        console.log(`${value} is an odd number.`);
+    }
+
+    console.log("-----------------------------");
+    console.log("Insert the next integer number.");
+    value = Number(prompt(">"));
```

Comment:
`value` was declared at line 4 as `const`, therefore should not be re-assigned. Please use `let` instead